ServiceSamples/ServiceTests/TLSTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using AuthenticationUtility;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using SoapUtility.UserSessionServiceReference;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+
+namespace ServiceTests
+{
+    [TestClass]
+    public class TLSTests
+    {
+        public static string sessionUrl = "/api/services/UserSessionService/AifUserSessionService/GetUserSessionInfo";
+
+        [TestMethod]
+        public void TLSAuthTest()
+        {
+            string GetUserSessionOperationPath = string.Format("{0}{1}", ClientConfiguration.Default.UriString.TrimEnd('/'), sessionUrl);
+            
+            // Creates an HttpWebRequest for user session URL
+            HttpWebRequest aadRequest = (HttpWebRequest)WebRequest.Create(GetUserSessionOperationPath);
+
+            // Change TLS version of HTTP request if the TLS version value is defined in ClientConfiguration
+            if (!string.IsNullOrWhiteSpace(ClientConfiguration.OneBox.TLSVersion))
+            {
+                aadRequest.ProtocolVersion = Version.Parse(ClientConfiguration.OneBox.TLSVersion);
+            }
+
+            string tlsRequestVersion = aadRequest.ProtocolVersion.ToString();
+            Console.WriteLine("The TLS protocol version for the HTTP request is {0}.", tlsRequestVersion);
```

Comment:
Please remove all WriteLine() statements. We might not want them in tests.

Reply:
Writeline will print out the log in the console. I don't see the problem with writeline in the test.
Also this sample is not only for test and but also for debugging purpose. Plus JsonTests also have writeline.

Reply:
Sure, let's keep it then.